app/mailers/user_mailer.rb

```diff
@@ -90,7 +90,8 @@ def claim_notification(creator_id, claimed_work_ids, is_user=false)
   
   # Sends a batched subscription notification
   def batch_subscription_notification(subscription_id, entries)
-    @subscription = Subscription.find(subscription_id)
+    @subscription = Subscription.find_by_id(subscription_id)
```

Comment:
Could you add a comment above this that explains, roughly, `# Use find_by_id (not find) so it returns nil if there's no subscription by that id`?